src/app/service/constants/constants.ts

```diff
@@ -1,15 +1,73 @@
 export default Object.freeze({
+	DISCORD_ROLE_GUEST_PASS: 'Guest Pass',
```

Comment:
Where is this used?

Reply:
Hmm I don't remember using this. grep -r 'DISCORD_ROLE_GUEST_PASS' shows this is already defined in env and roleIds.ts. Please confirm, then I'll remove it.

Reply:
Yeah in .env it is defined with the ID of the role since the name "Guest Pass" can change at any time by anyone with access to discord settings. Better to remove it please

DISCORD_ROLE_GUEST_PASS=857683488131907624

Reply:
> Hmm I don't remember using this. grep -r 'DISCORD_ROLE_GUEST_PASS' shows this is already defined in env and roleIds.ts. Please confirm, then I'll remove it.

Wait did you say grep, are you coding all of this via terminal? Might I suggest an IDEA kind ser

Reply:
no no, I use sublime editor, just use terminal to find a string in file :)

Reply:
ok removed